WBooks/ViewController.swift

```diff
@@ -9,11 +9,14 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    
```

Comment:
Remove these extra spaces to comply with standard Swift style (no spaces after the start or before the end of classes, objects, enumerations, etc).